PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -100,15 +100,20 @@ IDD_PREFERENCE_SUB_DARKMODE DIALOGEX 0, 0, 455, 185
 STYLE DS_SETFONT | DS_FIXEDSYS | DS_CONTROL | WS_CHILD
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
-    CONTROL         "Enable &dark mode",IDC_CHECK_DARKMODE_ENABLE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,20,20,150,10
+    CONTROL         "Enable &dark mode",IDC_CHECK_DARKMODE_ENABLE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,20,20,100,10
     //LTEXT           "* Notepad++ must be restarted to take effect completely",IDC_STATIC_DARKMODE_WARNING,20,35,310,16
-    CONTROL         "Black tone",IDC_RADIO_DARKMODE_BLACK, "Button",BS_AUTORADIOBUTTON | WS_GROUP,35,40,150,10
-    CONTROL         "Red tone",IDC_RADIO_DARKMODE_RED,     "Button",BS_AUTORADIOBUTTON ,35,55,150,10
-    CONTROL         "Green tone",IDC_RADIO_DARKMODE_GREEN, "Button",BS_AUTORADIOBUTTON ,35,70,150,10
-    CONTROL         "Blue tone",IDC_RADIO_DARKMODE_BLUE,   "Button",BS_AUTORADIOBUTTON ,35,85,150,10
-    CONTROL         "Purple tone",IDC_RADIO_DARKMODE_PURPLE,   "Button",BS_AUTORADIOBUTTON ,35,100,150,10
-    CONTROL         "Cyan tone",IDC_RADIO_DARKMODE_CYAN,   "Button",BS_AUTORADIOBUTTON ,35,115,150,10
-    CONTROL         "Olive tone",IDC_RADIO_DARKMODE_OLIVE,   "Button",BS_AUTORADIOBUTTON ,35,130,150,10
+    CONTROL         "Black tone",IDC_RADIO_DARKMODE_BLACK,  "Button",BS_AUTORADIOBUTTON | WS_GROUP,35,40,100,10
+    CONTROL         "Red tone",IDC_RADIO_DARKMODE_RED,      "Button",BS_AUTORADIOBUTTON,35,55,100,10
+    CONTROL         "Green tone",IDC_RADIO_DARKMODE_GREEN,  "Button",BS_AUTORADIOBUTTON,35,70,100,10
+    CONTROL         "Blue tone",IDC_RADIO_DARKMODE_BLUE,    "Button",BS_AUTORADIOBUTTON,35,85,100,10
+    CONTROL         "Purple tone",IDC_RADIO_DARKMODE_PURPLE,"Button",BS_AUTORADIOBUTTON,35,100,100,10
+    CONTROL         "Cyan tone",IDC_RADIO_DARKMODE_CYAN,    "Button",BS_AUTORADIOBUTTON,35,115,100,10
+    CONTROL         "Olive tone",IDC_RADIO_DARKMODE_OLIVE,  "Button",BS_AUTORADIOBUTTON,35,130,100,10
+
+    GROUPBOX        "Treeview Style",IDC_DARKMODE_TV_GB_STATIC,165,20,105,65,BS_CENTER
+    CONTROL         "Classic",IDC_RADIO_DARKMODE_TVCLASSIC, "Button",BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP,175,35,80,10
+    CONTROL         "Light",IDC_RADIO_DARKMODE_TVLIGHT,     "Button",BS_AUTORADIOBUTTON,175,50,80,10
+    CONTROL         "Dark",IDC_RADIO_DARKMODE_TVDARK,       "Button",BS_AUTORADIOBUTTON,175,65,80,10

```

Comment:
Treeview style cannot be disassociated from dark mode, not only it could confuse users, but also for preventing from the following case:
 
![image](https://user-images.githubusercontent.com/90293/125194505-f1175a80-e251-11eb-8f92-33700b9ce965.png)

The correct behaviour would be: when dark mode is enabled, treeview will be switched to "Dark" option automatically; When dark mode is disabled, treeview will be switched to "Light" or "Classic" (I guess "Light" looks better, but it's up to you).

Reply:
I can make it automatic,

but the issue here is that user can use dark theme with light mode and vice versa
![image](https://user-images.githubusercontent.com/55940305/125196193-70079580-e248-11eb-898a-e0a61004084a.png)
This was mentioned in https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10061

Would it be better if it is moved in style configurator

Reply:
Let's keep KISS - Dark option belongs to the dark mode.

Reply:
@donho 
> Let's keep KISS principle - Dark option belongs to the dark mode.

There might be someone who want dark mode but use light theme.
![image](https://user-images.githubusercontent.com/55940305/125199294-148fd480-e255-11eb-92dd-1856ac989034.png)

So should I remove this treeview style setting from preferences to make it hidden option (so it can be changed only via editing config.xml).

Reply:
Treeview style GUI option has to be removed for sure.
Hidden option is not necessary in my opinion as well.
KISS should not be applied to GUI, but to the source code as well.

Reply:
> Treeview style GUI option has to be removed for sure.
> Hidden option is not necessary in my opinion as well.
> KISS should not be applied to GUI, but to the source code as well.

Ok so should I keep only classic style, scrollbar fixes and enhancement to dark tone switching.

Reply:
Ok I will do commit later, some IRL stuff.